apps/core/scenes/bookingDetail/sectorDetails/SectorDetails.js

```diff
@@ -27,18 +29,26 @@ function SectorDetails(props: Props) {
         returnComponent={<SectorInfoReturn data={props.data} />}
         multicityComponent={<SectorInfoMulticity data={props.data} />}
       />
+      <Separator style={styles.bottomSeparator} />
+      <SegmentContainer />
     </Card>
   );
 }
 
 const styles = StyleSheet.create({
   card: {
     paddingHorizontal: 18,
+    borderBottomWidth: StyleSheet.hairlineWidth,
+    borderBottomColor: defaultTokens.paletteInkLighter,
   },
   separator: {
     marginTop: 7,
     marginBottom: 17.5,
   },
+  bottomSeparator: {
+    marginHorizontal: -18,
```

Comment:
Why do you need this? Could you please add some screenshot how it looks like? It could be more clear for me.

Reply:
Because the card component has a paddingHorizontal of 18, and the seperator is from screen start -> screen end

Reply:
[Figma](https://www.figma.com/file/BNs81bMp3VVuxqKIICbnNvqd/MMB?node-id=1%3A193)